pkgs/by-name/go/gopass-jsonapi/package.nix

```diff
@@ -42,50 +87,28 @@ buildGoModule rec {
     "-X main.commit=${src.rev}"
   ];
 
-  postFixup = ''
-    wrapProgram $out/bin/gopass-jsonapi \
-      --prefix PATH : "${gopass.wrapperPath}"
-
+  postInstall = ''
     # Generate native messaging manifests for Chrome and Firefox.
-    # It would be nicer to not do that in the fixupPhase but `gopass-jsonapi configure`
-    # is run which relies on on the wrapped `gopass-jsonapi`.
     export HOME=$(mktemp -d)
     ${gnupg}/bin/gpg --batch --passphrase "" --quick-generate-key "user <user@localhost>"
     ${gopass}/bin/gopass setup --name "user" --email "user@localhost"
+    ${lib.concatMapStrings (browser: createNativeMessagingManifest { inherit browser; }) [
+      "firefox"
+      "chrome"
+      "chromium"
+      "brave"
+      "vivaldi"
+      "iridium"
+      "slimjet"
+    ]}
```

Comment:
Use `builtins.attrNames manifestPaths` to not manually copy this list of browsers.